app/src/main/res/values/faqs.xml

```diff
@@ -3,10 +3,19 @@
+  <string name="faq_question_4">What is Oppia?</string>
+  <string name="faq_question_5">Who is an Administrator?</string>
+  <string name="faq_question_6">Why is the Exploration player not loading?</string>
+  <string name="faq_question_7">Why is my audio not playing?</string>
+  <string name="faq_question_8">How do I download a Topic?</string>
+  <string name="faq_question_9">I can\'t find my question here. What now?</string>
+  <string name="faq_answer_1">\u003cp\u003eIf it is your first time creating a profile and not have a PIN:\u003cbr\u003e\n1. From the Profile Chooser, tap on \u003cstrong\u003eSet up Multiple Profile\u003c/strong\u003es.\u003cbr\u003e\n2. Create a PIN and \u003cstrong\u003eSave\u003c/strong\u003e.\u003cbr\u003e\n3. Fill in all fields for the profile.\u003cbr\u003e\n\u0026nbsp; a. (Optional) Upload a photo.\u003cbr\u003e\n\u0026nbsp; b. Enter a name.\u003cbr\u003e\n\u0026nbsp; c. (Optional) Assign a 3-digit PIN.\u003cbr\u003e\n4. Tap \u003cstrong\u003eCreate\u003c/strong\u003e. This profile is added to your Profile Chooser!\u003cbr\u003e\n\u003cbr\u003e\nIf you have created a profile before and have a PIN:\u003cbr\u003e\n1. From the Profile Chooser, tap on \u003cstrong\u003eAdd Profile\u003c/strong\u003e.\u003cbr\u003e\n2. Enter your PIN and tap \u003cstrong\u003eSubmit\u003c/strong\u003e.\u003cbr\u003e\n3. Fill in all fields for the profile.\u003cbr\u003e\n\u0026nbsp; a. (Optional) Upload a photo.\u003cbr\u003e\n\u0026nbsp; b. Enter a name.\u003cbr\u003e\n\u0026nbsp; c. (Optional) Assign a 3-digit PIN.\u003cbr\u003e\n4. Tap \u003cstrong\u003eCreate\u003c/strong\u003e. This profile is added to your Profile Chooser!\u003cbr\u003e\n\u003cbr\u003e\nNote: Only the \u003cu\u003eAdministrator\u003c/u\u003e is able to manage profiles.\u003c/p\u003e</string>
+  <string name="faq_answer_2">\u003cp\u003eOnce a profile is deleted:\u003cbr\u003e\n- The profile cannot be recovered.\u003cbr\u003e\n- Profile information such as name, photos, and progress will be permanently deleted.\u003cbr\u003e\n\u003cbr\u003e\nTo delete a profile (excluding the \u003cu\u003eAdministrator\'s\u003c/u\u003e):\u003cbr\u003e\n1. From the Administrator\'s Home Page, tap on the menu button on the top left.\u003cbr\u003e\n2. Tap on \u003cstrong\u003eAdministrator Controls\u003c/strong\u003e.\u003cbr\u003e\n3. Tap on \u003cstrong\u003eEdit Profiles\u003c/strong\u003e.\u003cbr\u003e\n4. Tap on the Profile you would like to delete.\u003cbr\u003e\n5. At the bottom of the screen, tap \u003cstrong\u003eProfile Deletion\u003c/strong\u003e.\u003cbr\u003e\n6. Tap \u003cstrong\u003eDelete\u003c/strong\u003e to confirm deletion.\u003cbr\u003e\n\u003cbr\u003e\nNote: Only the \u003cu\u003eAdministrator\u003c/u\u003e is able to manage profiles.\u003c/p\u003e</string>
```

Comment:
In mock its showing  "- The profile cannot be recovered." as 
- The profile cannot be recovered. also please compare with mock the other bulleted  points..